### DISCUSSION TITLE
Well, guess it's time to install Linux - Ubuntu the best for this?

### DISCUSSION THREAD

**Discussion by Salsaloon** [madMAx43v3r/chia-plotter#117] [Q&A] [answered]
Looks like this works best on Linux so going to get an install Linux on my server. Some people have mentioned Ubuntu is the best for that, is that what others think too?

**Comment by SebMoore** [comment, marked as answer]
It's the easiest to use. You may have just started a war of preferences however 😂

**Reply by Salsaloon** [reply]
I saw some issues with Centos so was going to avoid that. Will just start with Ubuntu for now.

**Comment by delucca** [comment]
If you don't want to install Linux you could run it in a Docker container, following #102 

But, Linux is pretty good :) If you want to try it just do it ! Ubuntu is the easiest to use.

**Reply by Hardcore-fs** [reply]
Centos 7&8 is FINE!!!, the difficult one is osx pre 14.X

been  generating plots that validate & mine.
just stay on the cli for the best speed.
It's all about environment.....

Just throw in a spare HD into your computer, to keep it completely separate
don't mess with docker or vm's, you will loose several % VM ware spends it's time waiting for threads to execute

**Reply by Salsaloon** [reply]
It's good to know it works on Centos, thanks. I have a Poweredge R720 which is fully supported with Redhat and I suppose some of the packages would be easier to install with Centos. Although I did see Ubuntu certification for the Poweredge R720 too. But I'm going to do this for one purpose only and that's plotting so I guess which ever one works will be fine.

**Comment by Hardcore-fs** [comment]
The only problem with older kit is the wattage........

I have a rack full of hp dl380, but they burn  1-2kw per hour.
since i have zero back on the chia for over 1 month... it's not worth it....
when the NW was at 9 and diff was at 500.
Already in south china & H.K you CANNOT find anything bigger than a 4TB  unless you want to double or triple price.
no SSD or  PCI flash.
People are throwing their monthly salary at it...
